.changelog/11774.txt

```diff
@@ -0,0 +1,12 @@
+```release-note:enhancement
+resource/aws_ssm_maintenance_window_task: Add `cloudwatch_config` argument
+```
+
+```release-note:enhancement
+resource/aws_ssm_maintenance_window_task: Add `document_version` argument
```

Comment:
``````suggestion
resource/aws_ssm_maintenance_window_task: Add `task_invocation_parameters` `run_command_parameters` block `cloudwatch_config` and `document_version` arguments
``````